### DISCUSSION TITLE
About Congeneric Product Fleet (Jetbrains)

### DISCUSSION THREAD

**Discussion by 0x2CA** [zed-industries/zed#6495] [Feature Requests]
I think the preview version of Fleet is so similar. I don’t know if you have used this product. I think this is a very strong competitor. I think it can be studied and studied.

**Comment by JosephTLyons** [comment]
Hello @0x2CA, is there anything in particular about Fleet that you are really liking that you wish Zed had?  I look into Fleet every so often, just to keep up with the product a bit, but I don't use it daily.  I'm interested to hear what things users are liking about it.

**Reply by shbfy** [reply]
I’ve used Fleet a lot. For me, Fleet does a few distinct things well:

1. Ability to inject Fleet into Docker containers. Really handy for repeatable development setups.
2. General Code Intelligence from their IntelliJ backend is very good
3. ‘Tool’ windows where you can populate windows with a given tool, such as Docker, Terminal, Source Control etc.
4. Remote development

Hope this is helpful!

**Comment by 0x2CA** [comment]
At this stage, the facts prove that pursuing the path of AI is more competitive.

[Starting December 22, 2025, Fleet will no longer be available for download. We are now building a new product focused on agentic development.](https://blog.jetbrains.com/fleet/2025/12/the-future-of-fleet/)
